### PR TITLE
0.12.1

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/examples/pull/284

cc @GuillaumeGomez, @sdroege

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Thanks!

File: src/cairo_interaction.rs [hunk 1]
```diff
@@ -25,16 +25,21 @@ impl GdkSurfaceExt for Surface {
 }
 
 pub trait GdkPixbufExt {
-    fn create_surface<W: IsA<Window>>(&self, scale: i32, for_window: &W) -> Option<Surface>;
+    fn create_surface<W: IsA<Window>>(&self, scale: i32, for_window: Option<&W>)
```
Comment (sdroege) [inline src/cairo_interaction.rs]: This changes API and should not be in the bugfix release

CC @EPashkin @GuillaumeGomez

Comment (EPashkin) [reply src/cairo_interaction.rs]: My mistake,  I will add revert tomorrow morning

Comment (GuillaumeGomez) [reply src/cairo_interaction.rs]: Don't worry, I'm on it.

**Pull request merged**